week2/02_mlflow_experiment.py

```
@@ -0,0 +1,54 @@
+run_info = mlflow.get_run(run_id=f"{run_id}").to_dictionary()
+print(run_info)
+
+# COMMAND ----------
+with open("run_info.json", "w") as json_file:
+    json.dump(run_info, json_file, indent=4)
+
+# COMMAND ----------
+print(run_info["data"]["metrics"])
+
+# COMMAND ----------
+print(run_info["data"]["params"])
```

Ruff (0.7.0)

No newline at end of file

Add trailing newline

(W292)